bbb-common-web/src/main/scala/org/bigbluebutton/api2/meeting/OldMeetingMsgHdlrActor.scala

```diff
@@ -76,6 +78,14 @@ class OldMeetingMsgHdlrActor(val olgMsgGW: OldMessageReceivedGW)
 
   }
 
+  def handlePresenterUnassignedEvtMsg(msg: PresenterUnassignedEvtMsg): Unit = {
+    olgMsgGW.handle(new UserStatusChanged(msg.header.meetingId, msg.body.intId, "presenter", "false"))
```

Comment:
Please consider replacing `msg.body.intId` with `msg.body.presenterId`

Reply:
Ok, but then we must include this parameter in the common messages. https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bbb-common-message/src/main/scala/org/bigbluebutton/common2/msgs/PresentationMsgs.scala#L12

Reply:
Nvm then! I didn't see that `assignedBy` is separate. The idea was not to confuse presenter userId with userId of the user unsetting (possibly a moderator). The current code is OK!